samples/maps/demo/geoheatmap-earth-statistics/demo.js

```diff
@@ -0,0 +1,223 @@
+(async () => {
+
+    const topology = await fetch(
+            'https://code.highcharts.com/mapdata/custom/world-continents.topo.json'
+        ).then(response => response.json()),
+        landDayData = await fetch(
+            'https://cdn.jsdelivr.net/gh/highcharts/highcharts@a4b25b3845/samples/data/geoheatmap-land-sea-day-temp-august_2022.json'
+        ).then(response => response.json()),
+        data = [{
```

Comment:
Can we rename this consant to `datasets` to avoid confusion with the often-unsed `series.data` option?